### PR TITLE
Hop update

### PR DESCRIPTION
**Pull request by kelvinator07**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rsk-network-settings",
-  "version": "0.3.0-beta",
```
Comment (marcos-iov) [inline package.json:3]: Do we need to update the version? Only the readme changed

Comment (kelvinator07) [reply package.json]: True, Removed

**Review by nathanieliov — APPROVED** [review]
LGTM, but why the package-lock.json is being updated? Is it necessary?

**Comment by kelvinator07** [comment]
> LGTM, but why the package-lock.json is being updated? Is it necessary?

npm audit fix

**Pull request merged**